app-text/wordgrinder/wordgrinder-0.7.1.recipe

```diff
@@ -48,13 +53,34 @@ BUILD_PREREQUIRES="
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
+if [ -z "$secondaryArchSuffix" ]; then
+SUMMARY_dictionaries="Built-in dictionaries for WordGrinder"
+ARCHITECTURES_dictionaries="any"
+PROVIDES_dictionaries="
+	wordgrinder_dictionaries = $portVersion
+	"
+REQUIRES_dictionaries="
+	"
+fi
+
+
```

Comment:
We like to have only one line before defineDebugInfoPackage.
No need for 2 lines.

Reply:
I had do is